const getClothesById = async (req, res) => {
  try {
    
  } catch (error) {
    return res.status(404).json({ error: error });
  }
}

const getClothes = async (req, res) => {
  try {
    
  } catch (error) {
    return res.status(404).json({ error: error });
  }
}


export default { getClothesById, getClothes}